Gracefully handle eval() failure(s) for marker expressions (#7082)
pytest-dev/pytest: changelog/4583.bugfix.rst

> Prevent crashing and provide user friendly error(s) when marker expressions (-m) invoking of eval() raises a SyntaxError or TypeError

Reviewer: ```suggestion
Prevent crashing and provide a user-friendly error when a marker expression (-m) invoking of eval() raises
any exception.
```
